### ISSUE TITLE
Library incorrectly pads small packets

### ISSUE THREAD

**Issue by pthedinger** [xmos/lib_xtcp#23]
In xcoredev.xc, lines 100-103 it incorrectly uses 64 bytes as the minimum packet size and pads:
```
    if (len < 64)  {
      for (int i=len;i<64;i++)
        (uip_buf32, unsigned char[])[i] = 0;
      len=64;
    }
```
This should be 60, as the CRC will be automatically added to make the size up to the minimum frame size of 64 bytes.

**Comment by ahogen** [comment]
Fixed in commit ed83ea54059aecd5946d671ec2d7c1ca01fcd143

**Comment by samchesney** [comment]
Thank you @ahogen.

Closing as fix this is fixed on master.
